### ISSUE TITLE
cfc loss 

### ISSUE THREAD

**Issue by kashiani** [SiLangWHL/vMF-OP#1]
Thank you for your great work. I would be appreciated if you could update your code with loss_cfc loss function as paper.

**Comment by SiLangWHL** [comment]
Thanks for your attention. I will update some well-trained models and the loss_cfc codes today (UTC +8). Actually, I'm working on extending our ECCV paper to a journal paper.  So I will keep updating codes for more advanced performance.

**Comment by kashiani** [comment]
Thank you very much. I look forward to running you full code and learning your method.
